config/adapter.go

```diff
@@ -9,25 +9,13 @@ import (
 )
 
 type Adapter struct {
-	Endpoint string `mapstructure:"endpoint"` // Required
-	// UserSyncURL is the URL returned by /cookie_sync for this Bidder. It is _usually_ optional.
```

Comment:
Do we want to save the template documentation?

Reply:
This has been moved to `config/bidderinfo.go` for the `SyncerEndpoint` type.